### PR TITLE
add search functionality but have some issues

### PR DESCRIPTION
**Pull request by youko509**
li mache men sim fon recherche sam pa ekri anyen li banm tout kou ki sou li yo

### CODE REVIEW

**Review by undisplay — COMMENTED** [review]
Ajouter method clean lan nan form lan wap ka fe validasyon ladan'l,fe rechèch la

**Comment by youko509** [comment]
Ok mpral gad sa

File: courses/views.py [hunk 1]
```diff
@@ -26,4 +31,14 @@ def course_enrollement(request):
...
+
+
+class SearchResultsView(ListView):
+        
+    def get_queryset(self): 
+        query = self.request.GET.get('q')
+        object_list = Course.objects.annotate(
+        search = SearchVector('title')).filter(search=query)        
+        return  object_list
+
```
Comment (Chamane) [inline courses/views.py:44]: Rechèch lan mache selman si mwen tape tout mo an. Sa pa enteresan. Gade [la](https://stackoverflow.com/questions/49867384/django-searchvector-using-icontains) epi korije aspè saaa

Comment (Chamane) [reply courses/views.py:44]: Answit rechèch la pa fèt selman sou kou yo. Fè Freedy edew sou aspè saa

**Review by Chamane — CHANGES_REQUESTED** [review]

**Pull request closed without merging**